static/engine/WorldEditor.js

```diff
@@ -14,14 +14,26 @@ export class WorldEditor {
 			this.runAll.add(o)
 		})
 
-		mouse.addOnClick('paint', p => {
-			p.height = 100
-			p.width = 100
-			const o = new DynamicGameObject(new Position(p.x, p.y, p.width, p.height), 100, 100)
-
-			this.runAll.add(o)
-			ObjectPersistence.save(o)
-		})
+		mouse.addOnClick("paint", (p, c) => {
```

Comment:
don't do it like this. This is not a good solution

For now just don't worry about it drawing a picture when clicking a div